swift/ql/test/query-tests/Security/CWE-020/MissingRegexAnchor.expected

```diff
@@ -46,9 +46,6 @@
 | UnanchoredUrlRegex.swift:71:46:71:46 | https?://good.com | When this is used as a regular expression on a URL, it may match anywhere, and arbitrary hosts may come before or after it. |
 | UnanchoredUrlRegex.swift:78:39:78:39 | https?://good.com | When this is used as a regular expression on a URL, it may match anywhere, and arbitrary hosts may come before or after it. |
 | UnanchoredUrlRegex.swift:79:39:79:39 | https?://good.com:8080 | When this is used as a regular expression on a URL, it may match anywhere, and arbitrary hosts may come before or after it. |
-| UnanchoredUrlRegex.swift:82:3:82:3 | https?://good.com | When this is used as a regular expression on a URL, it may match anywhere, and arbitrary hosts may come before or after it. |
-| UnanchoredUrlRegex.swift:83:3:83:3 | https?:\\/\\/good.com | When this is used as a regular expression on a URL, it may match anywhere, and arbitrary hosts may come before or after it. |
-| UnanchoredUrlRegex.swift:84:3:84:3 | ^https?://good.com | This hostname pattern may match any domain name, as it is missing a '$' or '/' at the end. |
```

Comment:
Notably unlike the other test cases, these three were expressed as array initializers:
```
let trustedUrlRegexs = [
	"https?://good.com", // BAD (missing anchor), referenced below
	#"https?:\/\/good.com"#, // BAD (missing anchor), referenced below
	"^https?://good.com" // BAD (missing post-anchor), referenced below
]
```
Could it be we're not populating those correctly?

Reply:
Fixed by https://github.com/github/codeql/pull/18143 .